src/dataflow/onmt_helpers/create_onmt_prediction_report.py

```diff
@@ -45,17 +45,22 @@
 class OnmtPredictionReportDatum(PredictionReportDatum):
     datum_id: TurnId
     source: str
+    # The tokenized gold lispress.
     gold: str
+    # The tokenized predicted lispress.
     prediction: str
     program_execution_oracle: ProgramExecutionOracle
 
     @property
     def gold_canonical(self) -> str:
-        return try_round_trip(self.gold)
+        return to_canonical_form(self.gold)
```

Comment:
`try_round_trip` expects untokenized lispress str, but `gold` and `prediction` here are tokenized.
Also, this should not throw exceptions for `gold`, but for `prediction` we return `_PARSE_ERROR_LISPRESS` if it fails to parse.